analyzer/analyzer.go

```diff
@@ -63,7 +63,7 @@ func (r AnalyzeReturn) ConvertToResult(analyzerType, filePath string) *AnalysisR
 type analyzer interface {
 	Name() string
 	Analyze(content []byte) (AnalyzeReturn, error)
-	Required(filePath string, info os.FileInfo) bool
+	Required(filePath string, info os.FileInfo, inspectRemovedPackages bool) bool
```

Comment:
I don't think we have to add `inspectRemovedPackages` to all analyzers. At the moment, let's keep `file://` support in this PR and revert other changes. After merging this PR, we can take time to discuss `inspectRemovedPackages` in the dedicated PR. I'm sorry, but could you leave `file://` support only once?